assets/setup-wizard/style.scss

```diff
@@ -255,7 +255,9 @@ $page-background: #b7c9be;
 			max-width: 414px;
 
 			&--large {
-				max-width: 538px;
+				max-width: 558px;
```

Comment:
Since the text was updated, I also updated the width to continue with a nice positioning.